Orange/widgets/data/owtable.py

```diff
@@ -230,6 +231,8 @@ def __init__(self):
 
         gui.checkBox(box, self, "select_rows", "Select full rows",
                      callback=self._on_select_rows_changed)
+        gui.checkBox(box, self, "send_all_by_default", "Select all by default",
```

Comment:
This label is somewhat misleading. Rows are not selected by default. All rows are sent if none are selected. Though I don't like any of the following two selections much: "Send all data on empty selection", "Send all rows when none are selected".

What would happen if we don't have a checkbox at all, but instead always send all data? (If we have already discussed this, I surely hope the checkbox wasn't my suggestion. :)

Reply:
I'm down, no checkbox is even better. :)

Reply:
Agree, I was just about to say the checkbox is redundant.